cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -558,6 +562,17 @@ static DataFrom getDataFrom(HttpServletRequest request) {
     return dataFrom;
   }
 
+  /**
+   * Default: An empty list.
+   */
+  static List<String> getReplicaMovementStrategies(HttpServletRequest request) throws UnsupportedEncodingException {
```

Comment:
Duplicate code with `getGoals`, can we extract the common logic to a private function and use it from these functions?